src/resources/views/crud/fields/relationship/fetch_or_create.blade.php

```diff
@@ -101,50 +101,51 @@
         @if($activeInlineCreate)
             @include('crud::fields.relationship.inline_create_button', ['field' => $field])
         @endif
-    <select
-        name="{{ $field['name'].($field['multiple'] ? '[]' : '') }}"
-        data-field-is-inline="{{var_export($inlineCreate ?? false)}}"
-        data-original-name="{{ $field['name'] }}"
-        style="width: 100%"
-        data-force-select="{{ var_export($field['inline_create']['force_select']) }}"
-        data-init-function="bpFieldInitFetchOrCreateElement"
-        data-allows-null="{{var_export($field['allows_null'])}}"
-        data-dependencies="{{ isset($field['dependencies'])?json_encode(Arr::wrap($field['dependencies'])): json_encode([]) }}"
-        data-model-local-key="{{$crud->model->getKeyName()}}"
-        data-placeholder="{{ $field['placeholder'] }}"
-        data-data-source="{{ $field['data_source'] }}"
-        data-method="{{ $field['method'] ?? 'POST' }}"
-        data-minimum-input-length="{{ $field['minimum_input_length'] }}"
-        data-field-attribute="{{ $field['attribute'] }}"
-        data-connected-entity-key-name="{{ $connected_entity_key_name }}"
-        data-include-all-form-fields="{{ var_export($field['include_all_form_fields']) }}"
-        data-field-ajax="{{var_export($field['ajax'])}}"
-        data-inline-modal-class="{{ $field['inline_create']['modal_class'] }}"
-        data-app-current-lang="{{ app()->getLocale() }}"
-        data-include-main-form-fields="{{ is_bool($field['inline_create']['include_main_form_fields']) ? var_export($field['inline_create']['include_main_form_fields']) : $field['inline_create']['include_main_form_fields'] }}"
-        data-ajax-delay="{{ $field['delay'] }}"
-        data-language="{{ str_replace('_', '-', app()->getLocale()) }}"
-        data-debug="{{ config('app.debug') }}"
-
-        @if($activeInlineCreate)
-            @include('crud::fields.relationship.field_attributes')
-        @endif
-
-        @include('crud::fields.inc.attributes', ['default_class' =>  'form-control select2_field'])
-
-        @if($field['multiple'])
-        multiple
-        @endif
-    >
-        @if (!empty($current_value))
-            @foreach ($current_value as $key => $item)
-                <option value="{{ $key }}" selected>
-                    {{ $item }}
-                </option>
-            @endforeach
-        @endif
-
-    </select>
+        <input type="hidden" name="{{ $field['name'] }}" value="" @if(in_array('disabled', $field['attributes'] ?? [])) disabled @endif />    
+        <select
```

Comment:
Am I correc that nothing in this SELECT changed? It's just the input above that's new? I can't for the life of me spot any differences, it looks like a diff mistake right?

Reply:
Indeed, only the hidden input were added and also indentation for the `<select>` tag, I guess that is what's detected as a change, the identation. 

About the hidden input it should not be here in this PR, but I was about to submit it in a separate one so you can test it out. 

But it's easy to reproduce:
- go to `relationship_select` and remove the hidden input. Add a multiple relationship WITHOUT ajax to use that field. 
- Select one or more and save.
- go edit the entry and clear the select input by using the `x` in each individual option, not the `clear` button.
- save and you will see nothing happens (because when cleared it would not post NOTHING so will not be present in the request.) - IT WORKS with the clear button tho. 

I will be removing this from here and create a separate PR, maybe we can fix it in JS on the field side and don't need to add the hidden inputs. Thing is, with hidden input it works.

Reply:
Oh I get it, thanks. Yes, separate PR would be better, please 🙏

Reply:
I see you've already done that so feel free to click Resolve on this.